pkg/platform/yaml/utils.go

```diff
@@ -74,7 +74,6 @@ func Flatten(j interface{}) map[string][]string {
 			}
 		}
 	default:
-		fmt.Println(Emoji, "found invalid value in json", j, x.Kind())
```

Comment:
We have to provide some error statement here right and error handling if the kind is not handled in the switch case

Reply:
the work around this is pass logger in 3 layers